cmd/fleet/handleCheckin.go

```diff
@@ -38,8 +38,8 @@ var (
 	ErrAgentNotFound  = errors.New("agent not found")
 	ErrAgentCorrupted = errors.New("agent record corrupted")
 
-	kCheckinTimeout  = env.CheckinTimeout(30)   // 30s
-	kLongPollTimeout = env.LongPollTimeout(300) // 5m
+	kCheckinTimeout  = 30 * time.Second
+	kLongPollTimeout = 300 * time.Second // 5m
```

Comment:
hardcoded now?

Reply:
Being they are not configurable in the Elastic Agent, I just don't see the need to make it configurable. I think its better to not give so many options that can cause issues if a user configures it incorrectly.